libs/wallet-adapter/data-access/README.md

```diff
@@ -1,7 +1,239 @@
-# wallet-adapter-data-access
+# Angular Solana Wallet Adapter
 
-This library was generated with [Nx](https://nx.dev).
+This is the solana wallet adapter library for Angular
 
-## Running unit tests
+## Pre-requisites
+```
+@angular/core
```

Comment:
Just a nitpick: Would you mind adding the package version? Just for the sake of explicitness like this:

```
"@angular/core": "13.2.2",
"@heavy-duty/rxjs": "0.1.0",
"rxjs": "7.5.2",
"@ngrx/component-store": "13.0.2",
"@solana/web3.js": "1.36.0",
"@solana/wallet-adapter-base": "0.9.5"
"@heavy-duty/wallet-adapter": "0.2.2"
```

Reply:
Done here 19ba93f67c9f4027198f5e33b08c316fc5f3361f